ceremonies/src/tests.rs

```diff
@@ -1452,15 +1452,15 @@ fn endorse_newbie_fails_if_already_endorsed_in_previous_ceremony() {
 }
 
 #[test]
-fn endorse_newbie_fails_if_not_authorized() {
+fn endorse_newbie_fails_without_reputation() {
```

Comment:
well, bootstrappers initially have no reputation, so the naming might mislead.

Reply:
Yes, I renamed it: `endorse_newbie_fails_if_sender_has_no_reputation_and_is_not_bootstrapper`